Makefile

```diff
@@ -139,7 +138,6 @@ install-examples:
 	install -m 0644 support/demo.crt $(prefix)$(docexamplesdir)/
 
 uninstall: uninstall-bin \
-	uninstall-conf \
```

Comment:
This should still be here, no? The target is still intact.

Reply:
Yes, sorry. An leak from the commit amending I did.